packages/node/__dev__/test-api.ts

```diff
@@ -15,9 +15,9 @@ yargs(hideBin(process.argv))
     '$0',
     'Test API calls',
     {
-      'endpoint-name': {
+      'endpoint-id': {
         alias: 'e',
-        description: 'Endpoint name',
+        description: 'Endpoint id',
```

Comment:
I would expand on this description. As a user, when I try to view the help/description of a command and it returns the same string as the key, I find it kinda annoying.

Maybe something like:

```suggestion
        description: 'The ID of the endpoint to be tested, which is derived from the OIS title and endpoint name',
```